javascript/extractor/src/com/semmle/js/extractor/ExtractorConfig.java

```diff
@@ -51,7 +51,7 @@ public static ECMAVersion of(int version) {
     };
 
     public static enum SourceType {
-        SCRIPT, MODULE, AUTO;
+        SCRIPT, ES6_MODULE, CLOSURE_MODULE, AUTO;
```

Comment:
This new source type needs to be documented.

Reply:
Also, you cannot rename `MODULE` without introducing compensatory code in `Main` and `AutoBuild`, which rely on there being a source type called "module".

Reply:
Renamed it back to `MODULE` - that was a rat's nest I don't want to deal with now. I've left a note about it in the comment.